tests/src/JIT/HardwareIntrinsics/X86/Shared/Program.cs

```diff
@@ -21,17 +21,20 @@ public static int Main(string[] args)
 
             foreach (string testToRun in GetTestsToRun(args))
             {
-                Console.WriteLine($"Running {testToRun} test...");
+                TestLibrary.TestFramework.BeginTestCase(testToRun);
```

Comment:
This is the change that should fix the seed.

Reply:
That being said, this is conditioned on `WINCORESYS` being defined: https://github.com/dotnet/coreclr/blob/master/tests/src/Common/CoreCLRTestLibrary/TestFramework.cs#L29

Under what conditions is that not defined?

Reply:
(I ask since the current `DefineConstants` has it unconditionally set)

Reply:
@tannergooding As far as I know, the code in *TestFramefork.cs* should be updated such that
1.  `if WINCORESYS` is removed since we don't need it anymore
2. Default seed value is always set and could be altered if `CORECLR_SEED` is defined.
3. All `WINCORESYS` usages also should be removed from other tests

@BruceForstall It this correct?

Reply:
If we can come to an agreement here, I'll put up a separate PR to handle it.

Reply:
Ys, I agree with @echesakovMSFT . WINCORESYS should not be defined (although it is for just this TestFramework build)

Reply:
See https://github.com/dotnet/coreclr/pull/19041